### DISCUSSION TITLE
Hinting completion candidates

### DISCUSSION THREAD

**Discussion by ethan-leba** [qutebrowser/qutebrowser#6881] [Q&A] [answered]
I'm wondering if there's a way to select from completion candidates using hints, something like:

![Screen Shot 2021-12-10 at 3 33 05 PM](https://user-images.githubusercontent.com/4625599/145637957-28e2d652-994d-4c85-a8a8-f9e6940a7b0d.png)

Thanks for the great project!

**Comment by toofar** [comment, marked as answer]
There is not. I'm not sure there is an issue tracking a request for that either, the only similar one I remember is for jumping between categories.
Another option might be numbering the rows and having a go-to-count binding. Although that might not be as useful for people with small keyboards.
